src/routes/bitbucket/proxy/index.ts

```diff
@@ -182,6 +183,17 @@ export function proxyRoutes(runner: Runner, client: BitbucketClient) {
         targetBranch: prInfo.targetBranch,
       });
 
+      const priorityBranches = await StateService.getPriorityBranches();
+      const isPriorityBranch = validatePriorityBranch(priorityBranches, landRequest.prSourceBranch);
+      if (isPriorityBranch && request) {
+        await request.incrementPriority();
```

Comment:
This increments the priority of the request by 1 when landing. I guess it works assuming no other land requests in the queue are prioritised manually. What if there already are requests with priority modified in the queue? Or we are ok to give priority to those requests

Reply:
That's a good point - if land requests in the queue are manually prioritised first, then any other requests prioritised after will be behind them, but still prioritised before other requests. I feel like it's a bit of an edge case though, it's not often branches are manually prioritised and if they are they are, they're probably more urgent and one-offs?

Reply:
yeah fair enough